File: core/src/main/java/org/mifos/mobilewallet/core/data/fineract/entity/accounts/SavingAccountsListResponse.java
package org.mifos.mobilewallet.core.data.fineract.entity.accounts;

import java.util.ArrayList;
import java.util.List;

import org.mifos.mobilewallet.core.data.fineract.entity.accounts.savings.SavingAccount;

public class SavingAccountsListResponse {

    private List<SavingAccount> savingsAccounts = new ArrayList<>();

    public List<SavingAccount> getSavingsAccounts() {
        return savingsAccounts;
    }

    public void setAccounts(List<SavingAccount> savingsAccounts) {
        this.savingsAccounts = savingsAccounts;
    }
}

File: core/src/main/java/org/mifos/mobilewallet/core/data/fineract/entity/accounts/SavingAccountsListResponse.kt
package org.mifos.mobilewallet.core.data.fineract.entity.accounts

import org.mifos.mobilewallet.core.data.fineract.entity.accounts.savings.SavingAccount
import kotlin.collections.ArrayList

data class SavingAccountsListResponse(
        var savingsAccounts: List<SavingAccount>? = ArrayList())